### ISSUE TITLE
translate app into Korean

### ISSUE THREAD

**Issue by 0xadada** [mirai-audio/mir#43]
### Expected Behavior

Korean speakers can use the application in the Korean language.

### Task

All translations are located in https://github.com/mirai-audio/mir/tree/master/app/locales, with directory for each translation language. 

These should be added under a new `ko` directory in `app/locales`.

Use English as the source for translations, as it contains a complete list of up-to-date strings.
